scripts/load_chado_schema.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+usage() {
+    echo ""
+    echo "Usage: $0 -u <psql_user> -d <database_name> -h <host> -p <port> -s <chado prebuilt schema> [ -r ]"
+    echo ""
+    echo "Options:"
+    echo "  -u :          PostgreSQL username"
+    echo "  -d :          Name of the database to which the chado schema and ontologies are to be loaded"
+    echo "  -h :          Database host (default: localhost)"
+    echo "  -p :          Port (default: 5432)"
+    echo "  -s :          Chado schema to load (*sql.gz)"
+    echo "  -r :          Flag that triggers pg_dump if database already exists (optional)"
+    echo ""
+    exit
+}
+
+check_config() {
+    PSQL_CREATEDB=$(which createdb)
+    PSQL_DROPDB=$(which dropdb)
+    PSQL_EXEC=$(which psql)
+
+    if ! [ -x $PSQL_EXEC ] ; then
+        echo "You must install PostgreSQL and 'psql' must be accessible in current PATH"
+        exit
+    fi
+
+    if ! [ -x $PSQL_CREATEDB ] ; then
+        echo "'createdb' must be accessible in current PATH"
+        exit
+    fi
+
+    if ! [ -x $PSQL_DROPDB ] ; then
+        echo "'dropdb' must be accessible in current PATH"
+        exit
+    fi
+}
+
+load_chado_schema() {
+    file_type=`echo ${CHADO_SCHEMA##*.}`
+    echo "Loading chado schema ${CHADO_SCHEMA} to database '$2'."
+
+    if [ $file_type == "gz" ]; then
+        gunzip -c $CHADO_SCHEMA | psql -U $1 -d $2 -h $3 -p $4 &> $STDLOG
+    else
+        psql -U $1 -d $2 -h $3 -p $4 < $CHADO_SCHEMA &> $STDLOG
+    fi
+}
+
+dump_database() {
+    OUTPUT=$2"_database_dump_"$TIMESTAMP".sql"
+    echo pg_dump -U $1 -d $2 -h $3 -p $4 -f $OUTPUT -b
+    pg_dump -U $1 -d $2 -h $3 -p $4 -f $OUTPUT -b
+}
+
+# Default
+PORT="5432"
+HOST="localhost"
+PGDUMP=0
+TIMESTAMP=`date +%F_%T`
+
+STDLOG="load_chado_schema_${TIMESTAMP}.log"
+
+if [ $# -eq 0 ]; then
+    usage
+    exit
+fi
+
+while getopts "h:p:u:d:s:r" opt; do
+    case "$opt" in
+    h)  HOST=$OPTARG
+        ;;
+    p)  PORT=$OPTARG
+        ;;
+    u)  PG_USER=$OPTARG
+        ;;
+    d)  DB=$OPTARG
+        ;;
+    s)  CHADO_SCHEMA=$OPTARG
+        ;;
+    r)  PGDUMP=1
+        ;;
+    *)  usage
+        ;;
+    ?)  usage
+        ;;
+    esac
+done
+
+if [ -z "${PG_USER}" ] || [ -z "${DB}" ] || [ -z "${CHADO_SCHEMA}" ] ; then
+    usage
+fi
+
+check_config
+
+psql -U $PG_USER -h $HOST -p $PORT -c "CREATE DATABASE \"$DB\""
```

Comment:
@deepakunni3   This is the only thing that seemed to be creating problems.      I added an echo statement and I get this:

```
./scripts/load_chado_schema.sh -u nathandunn -d apollo-chado -h localhost -r -s 5432 -s chado-schema-with-ontologies.sql.gz
psql -U nathandunn -h localhost -p 5432 -c CREATE DATABASE "apollo-chado"
psql: FATAL:  database "nathandunn" does not exist
Cannot create database 'apollo-chado' due to improper connection parameters, lack of privileges or non-existent user 'nathandunn'.
```

However, I know this works this way:

```
createdb -U nathandunn apollo-chado 
```